assets/js/facility.js

```diff
@@ -96,7 +96,7 @@ export default function(channel){
     const {id, html} = payload;
     const $html = $(html)
     const existing = $('.js-cart').find(`*[data-stock-distribution-id="${id}"]`).first();
-    const quantity = intFromFindClass($html, '.js-quantity-requested')
+    const quantity = intFromFindClass($html, '.js-quantity-requested') || 0;
```

Comment:
May be cleaner to add this as fallback behavior in intFromFindClass since it would make behavior more consistent there

Reply:
I agree.
I put the fallback there because I wanted to avoid modifying existing code as much as possible.
I'll move the fallback behavior in intFromFindClass.

Reply:
Appreciate the thought, but it's a young app, plenty of opportunity to improve things, and if something breaks, well, we may learn something :-)  Obviously would be nice if test coverage gave more confidence, right now if only detects certain egregious breaks, hence a little manual playing before merging a PR, but a lots still in flux.  Also, if you know any local food banks/pantries in Quebec that might be interested and/or can help us translate into french, having a second language more usably translated may help figure out some of the patterns etc going forward, as it really needs to be very multi-language, but the current translation style from files probably won't work for that long term.